src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.ruleset

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Rules for Microsoft.Data.Sqlite" Description="Code analysis rules for Microsoft.Data.Sqlite." ToolsVersion="14.0">
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="SA1000" Action="None" />
```

Comment:
Rules have a default action. This disables most that are on by default and enables that one that's off by default. I'll see if there's a way to be explicit about which ones are enabled.